### ISSUE TITLE
improve wifi while using motioneyeos

### ISSUE THREAD

**Issue by kubilay-dev** [ccrisan/motioneyeos#3027]
hey, i recently launched motioneyeos on my private home router wirh wireless conmeczion on my raspberry 4 model b. it works perfectly fine but our wifi in the house becomes so slow that apps like youtube wont even load or our router crashes. is there any way to reduce the amount of wifi its taking? i was running it on 1280x720 on 30fps with video streaming (so other roommates can see the footage on their phones aswell).

i already thought about lowering the streaming quality and resolution and framerate but are there any more options for example in the expert settings which could help?

thanks in advance.

**Comment by starbasessd** [comment]
No, it is totally dependent on the quality/resolution, and framerate. I would drop framerate first, to 5 fps per camera. It has a stream from the camera to the motionEye hub, and the second (or third or fourth) stream to whoever is watching it. The bandwidth used adds up very fast.

**Comment by kubilay-dev** [comment]
Hm okay, but a 5 FPS camera seems really unreliable. Isn't there any way to maybe kill the quality for the framerate?

**Comment by starbasessd** [comment]
Sure, but then you are viewing blurred video like when they blur faces and plates on TV news reports... Go to Settings, Video Streaming, and reduce it there. You could also try reducing it with Resolution Dimmer in Preferences, but only works with rtsp and other high quality, not with simple mjpeg cameras like PiCams. You could also try the FrameRate dimmer, too. These do not affect the recording framerate or resolution.
